k8s-metadata-cleanup.tf

```diff
@@ -38,6 +38,9 @@ resource "kubernetes_deployment_v1" "beekeeper_metadata_cleanup" {
       metadata {
         labels = local.metadata_cleanup_label_name_instance
         annotations = {
+          "ad.datadoghq.com/${local.metadata_cleanup_full_name}.check_names": "[\"openmetrics\"]"
+          "ad.datadoghq.com/${local.metadata_cleanup_full_name}.init_configs": "[{}]"
+          "ad.datadoghq.com/${local.metadata_cleanup_full_name}.instances": "[{ \"prometheus_url\": \"http://%%host%%:9008/actuator/prometheus\", \"namespace\": \"${local.instance_alias}\", \"metrics\": [\"s3_bytes_deleted_bytes_total*\", \"hive_table_metadata_deleted_total*\",\"hive_partition_metadata_deleted_total*\",\"s3_paths_deleted_seconds_sum*\", \"s3_paths_deleted_seconds_count*\", \"metadata_cleanup_job_seconds_sum*\", \"hive_table_deleted_seconds_count*\", \"disk_*\", \"jvm*\", \"hikaricp*\"] }]"
```

Comment:
It would be better if the metrics are a variable list and you use this as default if you would like, so new metrics does not have to be a code submission.

Reply:
I converted it to a variable list. Thanks for the suggestion